dvc/command/diff.py

```diff
@@ -156,7 +155,7 @@ def add_parser(subparsers, parent_parser):
     )
     diff_parser.add_argument(
         "--checksums",
```

Comment:
Will need to rename `checksum` -> `hash` in the whole code base. Need to do that in a sync way to not introduce more confusion for developers and users.

Reply:
For the record, this question has been extracted to #3311